README.md

```diff
@@ -57,7 +57,7 @@ The easiest way to run the app is to use docker compose to create the service an
 The app requires: 
 * hmpps-auth - for authentication
 * redis - session store and token caching
-
+npm i
```

Comment:
was this change intended 🤔?

Reply:
no probably finger trouble - could you amend it back in your next PR?

Reply:
No problem - will do!